files/views.py

```
@@ -591,15 +599,49 @@ def edit_subtitle(request):
+            if subtitle_text:
+                try:
+                    with open(subtitle.subtitle_file.path, "w", encoding='utf-8') as ff:
+                        ff.write(subtitle_text)
+                    
+                    # Update media version even for invalid forms if text was provided
+                    subtitle.media.edit_date = timezone.now()
+                    subtitle.media.save(update_fields=['edit_date'])
+                    
+                    messages.add_message(request, messages.INFO, "Subtitle was edited")
+                    return HttpResponseRedirect(subtitle.media.get_absolute_url())
+                except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)